app/views/dashboard/_warn_node_removal_modal.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="modal fade warn-node-removal-modal" tabindex="-1" role="dialog" aria-labelledby="modal-label">
```

Comment:
is there a reason why we dont use slim on the modal?

Reply:
I copied and pasted from the other modal. We didn't have much time when I implemented that one and I wasn't aware of the html2slim converter. Lemme convert it.